index.bs

```diff
@@ -11092,8 +11142,16 @@ Note: The HTML Standard defines how a security check is performed. [[!HTML]]
             1.  If the argument at index |i| is declared with a [=optional argument/default value=],
                 then append to |values| that default value.
             1.  Otherwise, append to |values| the special value “missing”.
-        1.  Otherwise, append to |values| the result of [=converted to an IDL value|converting=]
-            |V| to IDL type |type|.
+        1. Otherwise:
+            1.  If |type| is an IDL type [=extended attribute associated with|associated with=] the
+                [{{StringContext}}] extended attribute, then set |V| to the result of performing
+                [=validate the string in context=], passing [=this=], |V|, the {{StringContext}}
+                extended attribute [=identifier=], and the [=identifier=]
+                of the [=operation=] or [=extended attribute=] of the first entry in |S|.
```

Comment:
"and the identifier of the operation or extended attribute of the first entry in S" is confusing. Is the operation's identifier or the extended attribute's identifier passed to "validate the string in context"?

Reply:
It's either other (https://whatpr.org/webidl/1392/5a2802b...36ddffb.html#dfn-effective-overload-set).

Reply:
Consider adding a note or feel free to resolve this 'conversation'.